src/screens/EPub/views/EditEPubChapter/ChapterEditor/ChapterContent.test.js

```diff
@@ -0,0 +1,49 @@
+import { render, screen } from '@testing-library/react';
+// TypeError: Cannot read properties of undefined (reading '__buildHTMLFromChapter')
+// (src/screens/EPub/controllers/file-builders/EPubParser.js:6:46)
+import { v4 as uuidv4 } from 'uuid';
+import ChapterContent from './ChapterContent';
+
+const test_uuid = '9b1deb4d-3b7d-4bad-9bdd-2b0d7b3dcb6d';
+
+jest.mock('uuid', () => {
```

Comment:
really good use of mock for a randomized ID from an external library. Once you merge this PR, you should definitely add this as an example test file in the [Testing wiki](https://github.com/classtranscribe/FrontEnd/wiki/Test-Framework-%E2%80%90-Jest---React-Testing-Framework)